packages/dx-react-grid/docs/guides/export.md

```diff
@@ -50,7 +50,7 @@ Note that the [plugin order](./plugin-overview.md#plugin-order) is important.
 
 4. Save the file
 
-   To handle file saving, implement the `onSave` callback of the `GridExporter` component. In this callback, save the file to the user's local storage. This can be done in many different ways. For example, you can use the `file-saver` package:
+   To save the file to the user's local storage, implement the `onSave` callback of the `GridExporter` component. In the following code, the file is saved using the `file-saver` package, but you can use any other package that provides similar functionality.
```

Comment:
Can we instead say:
To save the file to local storage at runtime, use the `onSave` callback for the `GridExporter` component. In the following code, the file is saved via the `file-saver` package, but you can use any other similar package.

Reply:
It's important that the local storage is the user's, not the developer's. In addition, the phrase "use the callback for the component" doesn't sound right. "**Implement** the callback for the component" does, so I suggest the following version:

```suggestion
   To save the file to the user's local storage at runtime, implement the `onSave` callback for the `GridExporter` component. In the following code, the file is saved via the `file-saver` package, but you can use any other similar package.
```

Reply:
I see. Then let's change it to:
To save a file to a user's local storage at runtime, implement the `onSave` callback for the `GridExporter` component.